app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -182,16 +183,44 @@ If you’re not eligible to use the new service, you may be able to:
 
+### Check the school you apply to can sponsor your visa
+
+Before you apply for a teaching job, contact the school to make sure it is a Home Office licensed visa sponsor. If the school is a sponsor, it can help you apply for your visa. Schools can also become sponsors to employ you. Submit your application, and if you are offered the job, you’ll then be able to apply for your visa.
+
 ### Your professional qualifications 
 
 If you haven't got QTS, schools you apply to will want to see alternative evidence of your teaching and academic qualifications. You can [get a statement of comparability from the UK information centre for international qualifications and skills](https://www.enic.org.uk/Qualifications/SOC/Default.aspx) showing how your qualifications compare to English ones.  
 
-You can search and apply for jobs by region, subject specialism and age group. Most teaching vacancies are advertised in March, April or the first half of May, with jobs usually starting at the beginning of the school year in September. 
+### Tips on applying
 
-Before you apply for a teaching job, contact the school to make sure it is a Home Office licensed visa sponsor. If the school is a sponsor, it can help you apply for your visa. Schools can also become sponsors to employ you. Submit your application, and if you are offered the job, you’ll then be able to apply for your visa.
+You can search and apply for jobs by region, subject specialism and age group. Most [teaching vacancies](https://teaching-vacancies.service.gov.uk/) are advertised in March, April or the first half of May, with jobs usually starting at the beginning of the school year in September. 
 
 You can set up a job alert to be updated about opportunities throughout the year.
 
+As part of your application, schools will usually ask for your:
+
+* educational and professional qualifications
+* employment history
+* personal statement
+* references
+
+Your personal statement needs to cover your teaching approach, your experience and how you would contribute to the life of the school. [Get tips on writing a personal statement](https://www.tes.com/jobs/careers-advice/application-and-interview/how-write-perfect-teaching-personal-statement).
+
+Some schools may ask for a curriculum vitae (CV) – [get tips on writing a CV](https://nationalcareers.service.gov.uk/careers-advice/cv-sections).
+
+If you’re shortlisted, you’ll be invited for interview and asked to demonstrate your teaching skills. 
+
+
+### Prepare for your interview
+
+You may be asked about your knowledge of English education in interviews for teaching jobs. To prepare, you can [read other teachers’ stories](/blog) and research the [English curriculum](https://www.gov.uk/national-curriculum) and [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards).
+
+### Safeguarding checks
```

Comment:
is there an extra space here as well? seems to be, on the review app

Reply:
No...

Reply:
It doesn't line up on the review app 
![Screenshot 2023-04-21 at 11 01 54](https://user-images.githubusercontent.com/117646221/233613367-d074bb3b-c791-4e8b-b01b-dc498a083fb2.png)